frontends/mit-open/src/pages/FieldPage/FieldSearchFacetDisplay.tsx

```diff
@@ -93,36 +89,46 @@ const AvailableFacetsDropdowns: React.FC<
 
   return (
     <>
-      {facetMap.map((facetSetting) => (
-        <FormControl key={facetSetting.name}>
-          <Select
-            multiple
-            displayEmpty
-            value={activeFacets[facetSetting.name as FacetKey] || []}
-            renderValue={() => {
-              return humanize(facetSetting.name)
-            }}
-            onChange={getHandleChangeForFacet(
-              activeFacets[facetSetting.name as FacetKey] || [],
-              facetSetting.name,
-            )}
-            sx={{ m: 1, minWidth: 140 }}
-          >
-            {filteredResultsWithLabels(
-              facetOptions(facetSetting.name) || [],
-              facetSetting.labelFunction || null,
-              constantSearchParams[facetSetting.name as FacetKey] || null,
-            ).map((facet) => (
-              <MenuItem
-                value={facet.key.toLowerCase()}
-                key={facetSetting.name.concat(":", facet.key)}
+      {facetMap.map((facetSetting) => {
+        const facetItems = filteredResultsWithLabels(
+          facetOptions(facetSetting.name) || [],
+          facetSetting.labelFunction || null,
+          constantSearchParams[facetSetting.name as FacetKey] || null,
+        )
+
+        return (
+          facetItems.length > 0 && (
+            <FormControl key={facetSetting.name}>
+              <Select
+                multiple
+                displayEmpty
+                value={activeFacets[facetSetting.name as FacetKey] || []}
+                renderValue={() => {
+                  return facetSetting.title
+                }}
```

Comment:
I did not include labels and values because I was working (roughly) off of this design
https://www.figma.com/proto/TL45iiHqKNdR8wEbQOtZyI/ferdi-pres-06-22?page-id=0%3A1&type=design&node-id=1-34352&viewport=355%2C384%2C0.07&t=NWYtxYilCOkgSeJg-1&scaling=scale-down&starting-point-node-id=1%3A34641

Showing the values looks weird if you select multiple options. I talked to Ferdi about other ways to display selected values and he said not to worry about it until we have an updated design